source/tunneling/SecureTunnelingContext.cpp

```diff
@@ -35,19 +35,12 @@ namespace Aws
 
                 SecureTunnelingContext::~SecureTunnelingContext()
                 {
-                    LOG_DEBUG(TAG, "SecureTunnelingContext::~SecureTunnelingContext");
-                    if (mSecureTunnel && mSecureTunnel->IsValid())
```

Comment:
We may need to preserve `mSecureTunnel->IsValid()`, based on the previous PR. Could move it to the Close() function in the wrapper.